_posts/2019-10-02-cockpit-204.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Cockpit 204
+author: pitti
+date: 2019-10-02
+tags: cockpit linux
+slug: cockpit-204
+category: release
+summary: Cockpit with Services and Machines improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.  Here are the release notes from version 204.
+
+### System: Highlight failed services
+
+The initial System page shows a "System Health" table entry if there are any
+failed systemd services:
+
+![Failed services on System page](/images/system-failed-services.png)
+
+The navigation menu will have a similar warning icon next to the "Services"
+entry in that case.
+
+The Services page now shows failed services at the top of the list, to make it
+easier to spot problems.
+
+### Machines: Configure read-only and shareable disks
+
+The "Disks" tab now shows whether a disk is read-only or shareable between
+multiple VMs:
+
+![Show shareable option in VM list](/images/machines-shareable-disk-list.png)
+
+With the new "Edit" button these properties can now be changed:
+
+![Edit shareable option](/images/machines-shareable-disk-edit.png)
+
+Sharing is only supported for disks in raw format. Note that sharing writable
+disks can easily destroy data when multiple VMs write to the disk at the same
+time; so use this feature with caution.
+
+Sharing will be enabled automatically when adding a disk to more than one VM.
+
+### Playground: Add index page
+
+The various "Playground" pages are meant for developers to test internal
```

Comment:
```suggestion
Various "Playground" pages are meant for developers to test internal
```

Reply:
I'm just removing a "The" here. I've been on a quest to try to make the beginning of paragraphs start with different words. (It's too easy to start every one with "The" and is a bit repetitive to read.)